recipe/meta.yaml

```diff
@@ -6,56 +6,59 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/calphy-{{ version }}.tar.gz
   sha256: 2e14ea2f090fe77bd4e191c59277be6452108efa9c70450704a3855adb2de584
   patches:
     - temp.patch
 
 build:
-  number: 1
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
   entry_points:
     - calphy = calphy.kernel:main
     - calphy_kernel = calphy.queuekernel:main
     - calphy_run_averaging = calphy.clitools:run_averaging
     - calphy_process_averaging = calphy.clitools:process_averaging
     - calphy_run_integration = calphy.clitools:run_integration
     - calphy_process_integration = calphy.clitools:process_integration
-            
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 2
+
 requirements:
   host:
     - python >=3.6
     - pip
-    - pytest-runner
   run:
     - python >=3.6
     - matplotlib-base
+    - pytest
+    - pyyaml
+    - mendeleev
     - tqdm
     - scipy
-    - numpy
-    - mendeleev
-    - pyscal
-    - pylammpsmpi
-    - pyyaml
-    - lammps >=2023.03.28
-    - pytest
-    - mpi4py
 
 test:
   imports:
     - calphy
-  requires:
-    - pip
   commands:
     - pip check
+    - calphy --help
+    - calphy_kernel --help
+    - calphy_run_averaging --help
+    - calphy_process_averaging --help
+    - calphy_run_integration --help
+    - calphy_process_integration --help
+  requires:
+    - pip
+    - lammps
+    - pyscal
+    - pylammpsmpi
```

Comment:
@srmnitc I tried to automatically update the dependencies, but `lammps`, `pyscal` and `pylammpsmpi` are not included in the `setup.py`. I can include `lammps` in `pylammpsmpi` then `pyscal` would be the only issue. Do you plan to publish `pyscal` on `pypi`?

Reply:
Ah there is pyscal on pypi. Its just called `pyscal2`.

Reply:
https://pypi.org/project/pyscal2/